x-pack/plugins/cross_cluster_replication/public/app/sections/follower_index_add/follower_index_add.js

```diff
@@ -70,20 +69,12 @@ export const FollowerIndexAdd = injectI18n(
                 );
               }
 
-              if (error) {
```

Comment:
With this removed, we now show a prompt to the user when there's an error:

![image](https://user-images.githubusercontent.com/1238659/51884823-bc431180-233d-11e9-834a-44ba04b798b7.png)

I'm a bit on the fence about this because it seems inaccurate and could possibly seem "deceptive" to the user. Imagine clicking the "Add remote cluster" button, completing the action, and then being redirected back here only to see this prompt again. At that point I'd be pretty surprised and confused -- there's something wrong but I wouldn't know what. What are your thoughts on this UX?

Reply:
Well, the previous UX was: the user enters follower indices, he clicks on "Add a follower index" and instead of accessing a form he had a red callout saying that the remote clusters could not be loaded. Unless he looks at the documentation and knows about the remote cluster field, that error message was not very helpful for the user, blocking him from accessing the form.

Also, if the list of clusters can't be loaded, he probably won't be able either to add a remote cluster. So that would help him understand better this message here in the add follower index form.